core/mock/goRoutineProcessorMock.go

```diff
@@ -0,0 +1,24 @@
+package mock
+
+import (
+	"bytes"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+)
+
+type GoRoutineHandlerMap = map[string]core.GoRoutineHandler
+
+type GoRoutineProcessorMock struct {
+	ProcessGoRoutineBufferCalled func(previousData map[string]GoRoutineHandlerMap, buffer *bytes.Buffer) map[string]GoRoutineHandlerMap
+}
+
+func (grpm *GoRoutineProcessorMock) ProcessGoRoutineBuffer(previousData map[string]GoRoutineHandlerMap, buffer *bytes.Buffer) map[string]GoRoutineHandlerMap {
```

Comment:
this won't implement core.GoRoutinesProcessor

Reply:
it's just an alias and should work